Cleans up ruins

Latentish opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
We as maptainers had a meeting on cleaning up the ruins in the game. As is our most spawned planets had the lowest quality ruins overall. Shiptest has inherited a lot of ruins that simply do not meet up to our newest standards, and are taking up the only ruin spawn on each planet. While multi-ruin spawning is in the pipeworks, we as maptainers have unanimously agreed to remove these maps.
This  PR also adds a new folder with maps that are being deprecated for either not working properly, or being too unbalanced to keep in the game. We want to see these maps reworked before being introduced back into the game. In this folder is a non-compiled .dm file that houses their datums for spawning, effectively removing them from spawning.
This also removes all content relating to the kinggoat. (not including achievements)
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
This PR removes a lot ruins that are either broken, wholly unbalanced, unfun, or poorly designed/mapped. This not only cuts down on these maps significantly, it also more evenly distributes the ratios of ruins per planet, which is inextricably tied to which planets spawn the most frequently, and how often. This has been a long time coming, but now it's been done carefully with the total approval by all 3 maptainers, and various admins and code maintainers.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:phoaly
add: Adds folder of deprecated ruins and datums.
del: Removes various low quality ruins, attached areas, and datums.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Latentish committed:
The whole ass pr

- Removes a number of maps not meeting up to shiptest standards
- Removes areas no longer in use
- Removes datums no longer in use
- Creates a deprecated folder and uncompiled .dm with all the datums of deprecated ruins therein
- Deprecates some maps that staff wishes to keep but cannot keep in game until they are fixed

ZephyrTFA committed:
Merge branch 'master' into Maptainer-spring-cleaning

Latentish committed:
Fixes issues with areas

Both of these maps used unique areas already in use by other maps that have been removed by this PR. Doing this in general is bad practice.

Latentish committed:
Merge branch 'Maptainer-spring-cleaning' of https://github.com/SweetBlueSylveon/Shiptest into Maptainer-spring-cleaning

Fikou reviewed (changes requested):
Delete the king goat ruin and all the code content (the king goat).

triplezeta commented:
> Delete the king goat ruin and all the code content (the king goat).

TRUE

tmtmtl30 commented:
death to the goat

Latentish committed:
Slaughters the Proverbial Lamb

- Removes all kinggoat content.
- No more goats of unusual rank
- Why was this so spread out
- Fuck megafauna in general
- Crusher achievements amirite?

Zevotech commented:
Jungle Witch was personally redone by me, I'd suggest giving its new layout a look and telling me what you think before removing it.

Latentish commented:
> Jungle Witch was personally redone by me, I'd suggest giving its new layout a look and telling me what you think before removing it.

A few reasons actually.
We came to the conclusion that while it isn't a horrible ruin, there's just not much to it aside from an enemy, some minor loot, and hazards in a square. I wouldn't be against having it set aside as a sub-ruin once multi-ruin generation happens, but it would need some work.
There's also this fun addition that made this decision a bit easier? I don't know if this is intended, or if it's a really bad coincidence. But this seems to communicate something something not good. A person, their cat, a sentience potion, and a love potion.
![image](https://user-images.githubusercontent.com/55075747/178056618-d67ed875-61cb-4d9e-97d7-49fa9407fbd8.png)

Fikou commented on code/datums/achievements/boss_achievements.dm:
```diff
@@ -66,12 +66,6 @@
-	name = "King Goat Killer"
-	desc = "The king is dead, long live the king!"
-	database_id = BOSS_MEDAL_KINGGOAT
-	icon = "goatboss"
```
there is no support of removing achievements iirc so bring these back

Latentish replied:
fixed

Fikou commented on code/modules/mob/living/simple_animal/hostile/megafauna/king_of_goats.dm:
```diff
@@ -1,343 +0,0 @@
-	desc = "The King of Kings, God amongst men, and your superior in every way."
-	icon_state = "king_goat2"
-	icon_living = "king_goat2"
-	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat/slab = 4, /obj/item/clothing/head/goatpelt/king = 1)
```
did you remove this goatpelt thing

Latentish replied:
yup

Zevotech commented:
> > Jungle Witch was personally redone by me, I'd suggest giving its new layout a look and telling me what you think before removing it.
> 
> A few reasons actually. We came to the conclusion that while it isn't a horrible ruin, there's just not much to it aside from an enemy, some minor loot, and hazards in a square. I wouldn't be against having it set aside as a sub-ruin once multi-ruin generation happens, but it would need some work. There's also this fun addition that made this decision a bit easier? I don't know if this is intended, or if it's a really bad coincidence. But this seems to communicate something something not good. A person, their cat, a sentience potion, and a love potion.

This Is Not The Remapped Version. This IS:
![image](https://user-images.githubusercontent.com/95449138/178078085-8549f310-be75-4596-bbf9-758bb12e85a4.png)
Way larger, has a decent amount of challenge and a reasonable reward.

Latentish committed:
Small patch

- Readds jungle witch pending a rework
- removes the last bit of goat content
- Readds the achievements for the goat.

Latentish committed:
Merge branch 'master' into Maptainer-spring-cleaning

Latentish commented:
> This Is Not The Remapped Version. This IS: ![image](https://user-images.githubusercontent.com/95449138/178078085-8549f310-be75-4596-bbf9-758bb12e85a4.png) Way larger, has a decent amount of challenge and a reasonable reward.

I've changed this.

Latentish committed:
Removes Jungle_witch changes

-Unrevertreverts the jungle_witch map so that it isn't touched.

MarkSuckerberg reviewed (approved):
Seems good to me :+1:
